app/controllers/authenticated_controller.rb

```diff
@@ -19,12 +20,6 @@ class AuthenticatedController < ApplicationController
     raise ActiveRecord::RecordNotFound.new
   end
 
```

Comment:
Is this the right call? Those familiar with PaperTrail would expect to see this in a controller or controller concern and not pushed down to a Helper? Do we really need access to this method in the views?